package.json

```diff
@@ -37,11 +37,11 @@
     "acorn-walk": "^8.0.0",
     "chalk": "^4.1.0",
     "commander": "^6.2.0",
-    "express": "^4.17.1",
     "filesize": "^6.1.0",
     "gzip-size": "^6.0.0",
     "lodash": "^4.17.20",
     "opener": "^1.5.2",
+    "serve-static": "^1.14.1",
```

Comment:
Seems like as per this comment:

https://github.com/webpack-contrib/webpack-bundle-analyzer/pull/212#discussion_r449811403

`serve-static` is a big reason why `express` is so large so if polka would work for our use case, it's much smaller than using `serve-static`.

Reply:
Oh, that other PR uses `sirv` for the static serving?

Reply:
Well, if you are ok to introduce sirv I will use it. I just remember there was a doubt to introduce polka packages.

Reply:
Oh, I don't really know what opinion @th0r has 😅 — sorry for being unclear. If things work then the size differences this provides might be useful?

As long as the server is still able to show new results when a bundle is changed like was the purpose behind introducing this feature back in https://github.com/webpack-contrib/webpack-bundle-analyzer/pull/74 then I would be OK with this change.

I think the supported node versions are good enough, too, and we don't accidentally regress from supporting node v10.13.0

Reply:
To support watch I passed "dev" flag which prevents caching.
I guess the size will be almost the same but without more polka dependencies.
https://github.com/webpack-contrib/webpack-bundle-analyzer/pull/212#issuecomment-735806643

Reply:
The `serve-static` is still in dependencies so I don't know how to figure out the size difference yet — some numbers on how this potentially affects the size of `node_modules` would be nice.

Reply:
https://packagephobia.com/result?p=static-serve
https://packagephobia.com/result?p=express (depends on static-serve out of the box)
https://packagephobia.com/result?p=sirv
https://packagephobia.com/result?p=polka (does not include sirv out of the box)
https://packagephobia.com/result?p=%40polka%2Furl (dependency of both sirv and polka)